### PR TITLE
docs: Update UserGuide

### LINKED ISSUE #95: [PE-D][Tester D] Deprecated command in example
The line `find Betsy` no longer works given the new implementation of the `find` command in Capital Connect.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: chengjunyuan/ped#6

### LINKED ISSUE #110: [PE-D][Tester A] Delay in displayed output for Key Employee section
There is a slight delay in the displayed output for the key employee section. 
So originally i have added a key employee ,Tim, in a startup company Allium with the description founder and email address tim@example.com.
After using the edit command to edit the employee Tim's description to CEO using , "edit-p 1 4 pd/CEO". It does not display immediately in the Key Employee section. It only appear if i click on the startup again.
As you can see in the picture below, although the command is successful and the command output says it changed the description to CEO, the person 4 tim, still retains the old description.

Although, this is not a serious flaw but it is good to make it update immediately after the command so as to prevent confusion for users.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: KiatLun/ped#6

### PR DESCRIPTION
**Pull request by menatahari**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -372,6 +380,23 @@ Examples:
...
 
+<box type="tip" seamless>
+
+**Tip:** Always click on the startup card after performing person-related operations to ensure you see the updated information.
+</box>
+
+### Clearing all entries : `clear`
+
+Clears all entries from CapitalConnect.
+
```
Comment (yeoshuheng) [inline docs/UserGuide.md:394]: Maybe it would be good to add a tip box warning on the execution of the clear command.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -99,6 +99,10 @@ startup investments!
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* Currently selected startup card will appear blue!
```
Comment (yeoshuheng) [inline docs/UserGuide.md]: Would it be good for some image or graphics to come together with this? I'm worried that solely saying "in blue" might be confusing for users, but I'm not too sure either.

**Review by yeoshuheng — CHANGES_REQUESTED** [review]
Some nits, sorry about that!

**Review by dwangwk — APPROVED** [review]
LGTM!

**Review by yeoshuheng — APPROVED** [review]
lgtm

**Pull request merged**